standalone-metastore/metastore-server/src/main/sql/hive/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.hive.sql

```diff
@@ -185,37 +185,36 @@ SELECT
   CASE WHEN CC_INITIATOR_ID IS NULL THEN cast (null as string) ELSE split(CC_INITIATOR_ID,"-")[size(split(CC_INITIATOR_ID,"-"))-1] END,
   CC_INITIATOR_VERSION,
   NULL,
-  CC_POOL_NAME,
+  NVL(CC_POOL_NAME, 'default'),
```

Comment:
is that fixed in schema?

Reply:
The original schema is illegal as Hive DDL, so I picked the one of `hive-schema-4.0.0-alpha-2.hive.sql`. I think it would give us a consistent view both when we build 4.0.0-alpha-2 from scratch and when we upgrade it from 4.0.0-alpha-1.
You can also see [this message](https://github.com/apache/hive/pull/5324#discussion_r1656249905).

Reply:
👍